README.md

```diff
@@ -12,6 +12,22 @@ The template provides the following project layout:
 - `example-client/client.py` Contains an example client that _uses_ the Managed Structure's API. This is useful for testing your Managed Structure locally but ultimately you will want to integrate your Managed Structure with your own application. 
 - `example-client/pyproject.toml` Contains the dependencies for the example client. Poetry is only used for the example client and is not used by the Managed Structure itself.
 
+## Running Managed Code without a Griptape Agent, Pipeline, or Workflow
+If your Python code uses a [Griptape Structure](https://docs.griptape.ai/stable/griptape-framework/structures/agents/)(such as an Agent, Pipeline, or Workflow), it will publish all of the appropriate events automatically when the Structure is run.
+
+However, if you want to run code that does _not_ rely on a Structure, you will need to publish events manually in order to communicate status back to the caller. For example, if you want to gracefully exit before running an Agent, you will need to manually publish an event:
```

Comment:
Should we not use "Agent" here, because Agents should automatically publish events properly? Maybe use an example without an agent?

Reply:
oh maybe this is being badly communicated by me. Here's the situation we will face:
1. You will want to run arbitrary code, like validating parameters, BEFORE invoking the Agent
2. If that arbitrary code needs to exit gracefully, it has to manually publish the `FinishStructureRunEvent` to tell the client "yo, I ended early."

Is there a better way to communicate this?

Reply:
ah, yeah I think the problem is the first paragraph. I'm expecting it to work because my code uses a griptape structure.. it's not clear that it _won't_ work if there's an error before the structure runs.

So maybe we need to clarify - once a structure has started running, then events will publish automatically. However, if you want to fire an event _before_ you run the Structure, or if you are executing code that doesn't rely on a Structure, you'll need to manually publish events.

Reply:
thank you for the clarity of thought that I was sorely lacking. Lemme get another rev